less/pages/encrypt/hybrid-test.less

```diff
@@ -0,0 +1,7 @@
+.encrypt.v3.remove-fields.video4 .page .videoSection .encrypt-signup .signupForm {
```

Comment:
This turns on the test. URL to test this locally is localhost:8080/encrypt/hybrid/4?test=remove-fields